static/js/impala/listing.js

```diff
@@ -32,9 +32,11 @@ function initListingCompat(domContext) {
         var $this = $(this);
         if ($this.find('.acr-override').length) {
             $this.addClass('acr');
-        } else if (!$this.hasClass('ignore-compatibility') &&
-                   $this.find('.concealed').length == $this.find('.button').length) {
-            $this.addClass('incompatible');
+        } else if (!$this.hasClass('ignore-compatibility') && (
+          $this.find('.concealed').length == $this.find('.button').length ||
+          $this.find('button.not-available').length
```

Comment:
I hate if-statements like that but don't mind me not knowing style-guides (if AMO has any) here but I'd usually do something like

```javascript
var isIncompatible = (
    !$this.hasClass('ignore-compatibility') &&
    $this.find('.concealed').length == $this.find('.button').length ||
    $this.find('button.not-available').length);

# ....

} else if isIncompatible
```

but that's just a nitpick.